fineract-provider/src/main/java/org/apache/fineract/cob/service/BusinessStepCategoryServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.cob.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import org.apache.fineract.cob.COBBusinessStep;
+import org.apache.fineract.cob.loan.LoanCOBBusinessStep;
+import org.springframework.data.util.Pair;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BusinessStepCategoryServiceImpl implements BusinessStepCategoryService {
+
+    private final List<Pair<String, Class<? extends COBBusinessStep>>> businessSteps = new ArrayList<>();
+
+    @PostConstruct
+    public void init() {
+        businessSteps.add(Pair.of("LOAN", LoanCOBBusinessStep.class));
```

Comment:
Can we extract the magic string into an enum rather or at least a constant somewhere? I wanna make sure people can reference it somewhere in a more type safe manner.